fix: `buffer` parameter should remain unchanged in WriteAsync

yiting2002 opened the pull request.

yiting2002 committed:
fix: `buffer` parameter should remain unchanged in WriteAsync

Joe4evr commented on src/Discord.Net.WebSocket/Audio/Streams/SodiumDecryptStream.cs:
```diff
@@ -12,28 +12,32 @@ public class SodiumDecryptStream : AudioOutStream
         private readonly byte[] _nonce;
+        private readonly byte[] _buffer;
 
-        public override bool CanRead => true;
-        public override bool CanSeek => false;
-        public override bool CanWrite => true;
```
Where did these properties go?

Joe4evr reviewed (changes requested):
Please explain the purpose of this PR.

yiting2002 committed:
feature: handle voice opcode 'Hello'

handles opcode 'Hello' to suppress warning.

use interval value from 'Hello' for voice heartbeat.
